### PR TITLE
Screenshot: Disable button if there is no test machine or not live.

### PR DESCRIPTION
**Pull request by jagtalon**
Here's a video: https://vid.me/eixe

### CODE REVIEW

**Comment by jagtalon** [comment]
For @MariagraziaAlastra

**Comment by MariagraziaAlastra** [comment]
Hey @jagtalon! It should become active when both the test machine and the example query are in place - if we have the test machine, but no example query, I guess we won't be able to generate a screenshot anyway :)

**Comment by jagtalon** [comment]
@MariagraziaAlastra ah good point!

**Comment by jagtalon** [comment]
@MariagraziaAlastra done. Screencast: https://vid.me/w3Hs (slow internet connection, though)

File: src/js/ia/IAPage.js [hunk 1]
```diff
@@ -240,9 +240,43 @@
...
+                        var $generate_screenshot = $(".generate-screenshot--button");
+                        if((test_machine && example_query) || ia_data.live.dev_milestone === "live") {
+                            $generate_screenshot.removeClass("generate-screenshot--disabled");
+                        } else {
+                            $generate_screenshot.addClass("generate-screenshot--disabled");
+                        }
+                    }
+
+                    enableScreenshotButton();
+                    $("body").on("change", ".test_machine", function(evt) {
```
Comment (MariagraziaAlastra) [inline src/js/ia/IAPage.js]: There's no need to create two new events for calling the enableScreenshotButton function: you can check inside the autocommit function (somewhere [here](https://github.com/duckduckgo/community-platform/blob/jag/disable-screenshot/src/js/ia/IAPage.js#L752)) if the saved field was either the test machine or the example query, and in that case, call the function.

**Comment by MariagraziaAlastra** [comment]
Thanks @jagtalon! :+1:

**Pull request merged**